clj-refactor.el

```diff
@@ -8,7 +8,7 @@
 ;;         Benedek Fazekas <benedek.fazekas@gmail.com>
 ;; Version: 2.3.0-SNAPSHOT
 ;; Keywords: convenience, clojure, cider
-;; Package-Requires: ((emacs "24.4") (s "1.8.0") (dash "2.4.0") (yasnippet "0.6.1") (paredit "24") (multiple-cursors "1.2.2") (cider "0.11.0") (edn "1.1.2") (inflections "2.3") (hydra "0.13.2"))
+;; Package-Requires: ((emacs "24.4") (s "1.8.0") (dash "2.4.0") (yasnippet "0.6.1") (paredit "24") (multiple-cursors "1.2.2") (clojure-mode "20160507") (cider "0.11.0") (edn "1.1.2") (inflections "2.3") (hydra "0.13.2"))
```

Comment:
that would block us merging this until clojure-mode is released. on the other hand I don't want this to trigger clojure-mode and cljr releases.

So I suppose let's do most of the elisp stuff (as discussed on gitter) and then cut a release of both clojure-mode and cljr.